selfdrive/ui/translations/main_nl.ts

```diff
@@ -0,0 +1,1305 @@
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="130"/>
+        <source>Regulatory</source>
+        <translation>Regulatie</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="130"/>
+        <source>VIEW</source>
+        <translation>BEKIJKEN</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="138"/>
+        <source>Change Language</source>
+        <translation>Taal Wijzigen</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="138"/>
+        <source>CHANGE</source>
+        <translation>WIJZIGEN</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="142"/>
+        <source>Select a language</source>
+        <translation>Selecteer een taal</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="162"/>
+        <source>Reboot</source>
+        <translation>Opnieuw Opstarten</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="167"/>
+        <source>Power Off</source>
+        <translation>Uitschakelen</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="187"/>
+        <source>openpilot requires the device to be mounted within 4° left or right and within 5° up or 8° down. openpilot is continuously calibrating, resetting is rarely required.</source>
+        <translation>openpilot vereist dat het apparaat binnen 4° links of rechts en binnen 5° omhoog of 8° omlaag wordt gemonteerd. openpilot kalibreert continu, resetten is zelden nodig.</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="198"/>
+        <source> Your device is pointed %1° %2 and %3° %4.</source>
+        <translation> Uw apparaat is gericht op %1° %2 en %3° %4.</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="199"/>
+        <source>down</source>
+        <translation>omlaag</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="199"/>
+        <source>up</source>
+        <translation>omhoog</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="200"/>
+        <source>left</source>
+        <translation>links</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="200"/>
+        <source>right</source>
+        <translation>rechts</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="211"/>
+        <source>Are you sure you want to reboot?</source>
+        <translation>Weet u zeker dat u het apparaat opnieuw wilt opstarten?</translation>
```

Comment:
```suggestion
        <translation>Weet u zeker dat u opnieuw wilt opstarten?</translation>
```

Reply:
Let's try to match the English as closely as possible.

(for other reviewers: the suggested translation means "Are you sure you want to reboot the device")